test/parallel/test-child-process-spawnsync-validation-errors.js

```diff
@@ -4,6 +4,12 @@ const assert = require('assert');
 const spawnSync = require('child_process').spawnSync;
 const signals = process.binding('constants').os.signals;
 
+const invalidArgTypeError =
+  common.expectsError({ code: 'ERR_INVALID_ARG_TYPE', type: TypeError }, 56);
```

Comment:
On Windows there are only 36 entries because a couple tests are not run on Windows.

Reply:
See also https://ci.nodejs.org/job/node-test-binary-windows/11411/COMPILED_BY=vs2015,RUNNER=win2008r2-vs2017,RUN_SUBSET=3/console